blacklistfilter/evaluator/evaluator.py

```diff
@@ -0,0 +1,316 @@
+import csv
+import json
+from queue import Queue
+import os
+import sys
+import signal
+import time
+from threading import Thread, Event
+import requests
+from datetime import datetime
+import pytrap
+import shutil
+from optparse import OptionParser
+from collections import defaultdict
+
+#Enter an API key to obtain informations from nerd
+NERD_API_KEY = None
```

Comment:
This should be loaded from configuration (see https://github.com/CESNET/Nemea-Detectors/tree/master/blacklistfilter/blacklist_downloader and `@BLACKLISTFILTERDIR@` as an example) . It is not a best practice to modify installed scripts.